### PR TITLE
oasis-std: Add ability to get AAD

### PR DESCRIPTION
**Pull request by abukosek**
This PR adds the necessary functions to access the additional authenticated data (AAD).

Related PR: https://github.com/oasislabs/oasis-parity/pull/154

### CODE REVIEW

File: oasis-std/src/backend/wasi.rs [hunk 1]
```diff
@@ -43,6 +43,10 @@ pub fn payer() -> Address {
     env_addr("PAYER")
 }
 
+pub fn aad() -> Vec<u8> {
+    hex::decode(&std::env::var_os("AAD").unwrap().into_vec()).unwrap()
```
Comment (nhynes) [inline oasis-std/src/backend/wasi.rs]: please, please anything but hex. base64, perhaps?

Comment (abukosek) [reply oasis-std/src/backend/wasi.rs]: OK :D
I'll fix the other PR too.

Comment (abukosek) [reply oasis-std/src/backend/wasi.rs]: Done! :)

**Comment by nhynes** [comment]
Ah, that's why it's not allowing merge. @abukosek, you'll need to GPG sign your commits in the future! There's no way this PR is malicious, though, so I'll merge it in manually :)

**Comment by abukosek** [comment]
Ah, oops :flushed: Thanks! :)

**Pull request merged**